static/skin/index.js

```diff
@@ -277,23 +277,10 @@
                 setTimeout(() => {
                     const divTag = document.createElement('div');
                     divTag.setAttribute('class', 'noResults');
-                    divTag.innerHTML = `No result. Would you like to <a href="/?lang=">reset filter</a>?`;
+                    const resetLink = window.location.pathname + '?lang=';
+                    divTag.innerHTML = `No result. Would you like to <a href="${resetLink}">reset filter</a>?`;
```

Comment:
Simply removing the '/' from the link would do the trick.

Reply:
Neat